.github/workflows/docker-test.yml

```diff
@@ -0,0 +1,38 @@
+on:
+    push:
+      branches:
+      - dev
+    pull_request:
+      types: [opened, synchronize, reopened]
+
+name: Update and Test Docker Image
+jobs:
+  publish:
+    name: Build and publish
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
I don't think we need to check for all python versions. One should be enough. Maybe stick to 3.8?

Reply:
I thought since we have release configured for 3.6,3.7,3.8 so I thought it would be better to check for them too. Alright, will get it patched up.

Reply:
We are anyways checking it on CLI. The whole idea of this is that during each PR, it shouldn't break the docker file.